microsoft-edge/progressive-web-apps-chromium/how-to/share.md

```diff
@@ -129,7 +129,7 @@ The Windows **Share** dialog is displayed.  The user picks an app in which to sh
 
 ![The Windows Share dialog allows the user to pick which app should receive the shared content](../media/devtools-tips-share.png)
 
-You can find the [source code on GitHub](https://github.com/captainbrosset/devtools-tips/).  The app uses the Web Share API in the [share.js](https://github.com/captainbrosset/devtools-tips/blob/main/src/assets/share.js#L38) source file.
+You can find the [source code on GitHub](https://github.com/captainbrosset/devtools-tips/).  <!-- The app uses the Web Share API in the [share.js](https://github.com/captainbrosset/devtools-tips/blob/main/src/assets/share.js#L38) source file.  todo: 404; fix destination or update this link -->
```

Comment:
@captainbrosset pls fix link destination
https://github.com/captainbrosset/devtools-tips/blob/main/src/assets/share.js#L38
then i can un-comment link

Reply:
The sharing functionality was removed from devtoolstips unfortunately. Instead, let's use the PWAmp demo in this section.
That means we need to update the whole "Demo of sharing content" section, e.g. instead of "DevTools Tips is a PWA that uses..." we should say "PWAmp is a demo PWA that uses ...".

Here are the steps to test in PWAmp:

1. Open PWAmp in Edge, and install it as a PWA
2. In the installed PWA, import a local audio file (by drag-dropping it onto the app window). If you don't have an audio file, there are 2 demo mp3s here: https://github.com/MicrosoftEdge/Demos/tree/main/pwamp/songs
3. Next to the newly imported song, there's a `...` action icon, click on it and select **Share**.

Once we've got this in the docs, we can change the github source code link to: https://github.com/MicrosoftEdge/Demos/blob/main/pwamp/app.js#L328-L341

Reply:
rewrote "Demo of sharing content" section/link @captainbrosset